internal/chezmoi/chezmoi.go

```diff
@@ -179,3 +179,11 @@ func mustTrimSuffix(s, suffix string) string {
 	}
 	return s[:len(s)-len(suffix)]
 }
+
+// ensureSuffix adds suffix to s if s is not suffixed by suffix.
+func ensureSuffix(s, suffix string) string {
+	if !strings.HasSuffix(s, suffix) {
```

Comment:
It's a tiny bit, but I prefer to use positive logic where possible, i.e. if `s` already has suffix `suffix` then return `s`.